ConsistencyManagerTests/PauseListenerTests.swift

```diff
@@ -11,7 +11,7 @@ import XCTest
 @testable import ConsistencyManager
 
 /**
- This test suite focuses on the pauseListeningForUpdates and resumeListeningForUpdates methods in ConsistencyManager.swift.
+ This test suite focuses on the pauseListener and resumeListener methods in ConsistencyManager.swift.
```

Comment:
tick marks for code in docs (`pauseListener`, `resumeListener`), here and elsewhere